File: Python3/Check if the Sentence Is Pangram.py
#1
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        c = {}
        for s in sentence:
            if s in c:
                pass
            else:
                c[s] = 1
        
        return len(c) == 26

#2
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        return len(set(sentence)) == 26

File: Golang/Check if the Sentence Is Pangram.go
//#1
func checkIfPangram(sentence string) bool {
    c := make(map[rune]int)
    for _, s := range sentence {
        if _, ok := c[s]; ok {
            
        } else {
            c[s] = 1
        }
    }
    return len(c) == 26
}